runtime/port/unix/j9sysinfo.c

```diff
@@ -1675,8 +1675,11 @@ j9sysinfo_get_cache_info(struct J9PortLibrary *portLibrary, const J9CacheInfoQue
 	) {
 		/* L1 data cache line size */
 		int32_t rc = (int32_t)sysconf(_SC_LEVEL1_DCACHE_LINESIZE);
-		if (rc >= 0) {
+		if (rc > 0) {
 			result = rc;
+		} else if (rc == 0) {
+			// Cache line size in unavailable on some systems
```

Comment:
Nitpick - we typically avoid `//` comments in VM code as not all supported compilers accept them and its easier to blanket ban them then ensure they are only present where in platform-specific code

```suggestion
			/* Cache line size in unavailable on some systems */
```

Reply:
Updated the code as suggested.